plugins/flashrom/flashrom.quirk

```diff
@@ -6,3 +6,8 @@ DeviceId=librem15v3
 [HwId=52b68c34-6b31-5ecc-8a5c-de37e666ccd5]
 DeviceId=LT1000
 VersionFormat=quad
+
+# Starlabs LabTop L4
+[HwId=baf1d04e-fd16-5e6a-93cc-1c23d171f879]
+DeviceId=StarlabsLabTopL4
+VersionFormat=triplet
```

Comment:
triplet is correct?  others in file use quad, so just want to make sure.

Reply:
In this case, yes. I've got a follow up patch after this PR to make the coreboot stuff more sane.